src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -15,133 +15,10 @@ namespace ov {
 namespace frontend {
 namespace tensorflow {
 namespace op {
-#define OP_CONVERTER(op) OutputVector op(const NodeContext& node)
-#define OP_T_CONVERTER(op) \
-    template <class T>     \
-    OutputVector op(const NodeContext& node)
-
-OP_T_CONVERTER(translate_unary_op);
-OP_T_CONVERTER(translate_binary_op);
-OP_T_CONVERTER(translate_direct_reduce_op);
-
-OP_CONVERTER(translate_add_n_op);
-OP_CONVERTER(translate_arg_max_op);
-OP_CONVERTER(translate_arg_min_op);
```

Comment:
please do not move this to hpp file because it affects build time much

Reply:
Otherwise, I cannot access them -- they are in cpp.